package.json

```diff
@@ -6,14 +6,17 @@
   "scripts": {
     "build": "sass scss/bitstyles.scss build/bitstyles.css --style=compressed --no-source-map",
     "postbuild": "postcss build/bitstyles.css --replace",
-    "lint:scss": "stylelint --ignore-path .gitignore 'scss/**/*.scss' --customSyntax postcss-scss && yarn prettier:scss",
-    "lint:js": "eslint --ignore-path .gitignore --ext js,jsx . && yarn prettier:js",
-    "lint:docs": "prettier --ignore-path .gitignore --check './**/*.{md,mdx}'",
-    "lint": "yarn lint:scss && yarn lint:js && yarn lint:docs",
+    "lint:scss": "stylelint --ignore-path .gitignore 'scss/**/*.scss' --customSyntax postcss-scss",
+    "postlint:scss": "yarn prettier:scss",
```

Comment:
Wow, neat! Did not know this existed 💡

Reply:
It useful! But it seems to run in a new process, so I’m not 100% sure about it — results can output to stdout after the main task has run. wdyt, should we stick with this, or use npm-run-all or something similar?

Reply:
Oh! So it runs in the background and then pops up out of the nowhere once it's done? That's not really useful then though 😞 

It's generally recommended to run formatting **before** linting as they can easier cancel each other out the other way.

So maybe the setup is just easiest to control if we keep it as it was: separate task for `prettier` and `linters` and then additional tasks that combine them 🤷 

`npm-run all` would do things in parallel, right? Maybe a nice addition on top 🍨 But probably not suuuper necessary as most people have the tools integrated in their editor setup anyways and then just use the tasks as a pre-flight check.

Reply:
thanks for the feedback! I removed the `post` hooks, and now the `lint:scss` and `prettier:scss`(and for js & mdx) tasks are back. So that the `--fix` option works, prettier is no longer run in those individual lint tasks, but it is when running the all-in-one `yarn lint`. Does that work, wdyt?